src/pages/sign-up-verification.page.js

```diff
@@ -29,16 +29,43 @@ export default class SignUpVerificationPage extends Component {
     this.state = {
```

Comment:
in general im a little unclear why signUp and signIn both need seperate pages. cant we combine them and make a few minor checks?

Reply:
Yeah can happen later...priority is to cut a prod tag and not rework it..I'll add it to the improvements list with Kev